11 Free online shopping.rb

```diff
@@ -0,0 +1,11 @@
+# Determines whether a shopping order is eligible for free shipping.
+# It qualifies if the total cost of items purchased exceeds $50.00.
+def free_shipping(shop_cart)
+  total = 0
+  shop_cart.each_value { |v| total += v } 
+  return total > 50 ? true : false

```

Comment:
This is all you need. It will return true or false already. 

```suggestion
  total > 50

```